specs/commandline/sign.md

```diff
@@ -116,3 +152,15 @@ Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(
 Resolved artifact tag `v1` to digest `sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9` before signing.
 Successfully signed localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
+
+### Sign an artifact and store the signature using OCI image manifest
+
+```shell
+notation sign --use-image-manifest <registry>/<repository>@<digest>
+```
+
+## References
+
+[oci-image-manifest]: https://github.com/opencontainers/image-spec/blob/v1.0.2/manifest.md
+[oci-backward-compatibility]: https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#backwards-compatibility
+[registry-support]: https://notaryproject.dev/docs/registrysupport/
```

Comment:
These lines are not rendered in the markdown page. Hence:
```suggestion
oci-image-manifest: https://github.com/opencontainers/image-spec/blob/v1.0.2/manifest.md</br>
oci-backward-compatibility: https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#backwards-compatibility</br>
registry-support: https://notaryproject.dev/docs/registrysupport/</br>
```

Reply:
I should keep the format, but remove the title `##Reference`, this links won't be showed, but replace the reference in the markdown file.